packages/utils/src/style-to-string.ts

```diff
@@ -1,7 +1,11 @@
 // https://stackoverflow.com/a/61410824
 
-export const styleToString = (style: any) => {
-  return Object.keys(style).reduce(
+export const styleToString = (style: React.CSSProperties | undefined) => {
```

Comment:
If we are using this function, we need to make sure that we're passing the style.

```suggestion
export const styleToString = (style: React.CSSProperties) => {
```

Reply:
style props for each component in packages/* could be undefined
if one use styleToString(props.style) directly

Reply:
thx! I will revise this part!